x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/Count.java

```diff
@@ -91,6 +94,9 @@ public DataType dataType() {
 
     @Override
     public AggregatorFunctionSupplier supplier(List<Integer> inputChannels) {
+        if (field() instanceof AggregateDoubleMetricSubAttribute) {
```

Comment:
I think we can remove this?

Reply:
Good suggestion Nhat. I was able to remove it.